.clang-format

```diff
@@ -111,4 +111,5 @@ SpacesInSquareBrackets: false
 Standard:        Cpp11
 TabWidth:        8
 UseTab:          Never
-...
+
+...
```

Comment:
This file should not be modified.

Reply:
I've made sure the file is restored to its previous state.

Reply:
```bash
$ git diff master imtiaz-branch .clang-format
```
> diff --git a/.clang-format b/.clang-format
> index 4b76204..f0ca9d3 100644
> --- a/.clang-format
> +++ b/.clang-format
> @@ -111,4 +111,5 @@ SpacesInSquareBrackets: false
>  Standard:        Cpp11
>  TabWidth:        8
>  UseTab:          Never
> -...
> +
> +...
> \ No newline at end of file